domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -77,6 +79,33 @@ object InteractionObjectTestBuilder {
     ).build()
   }
 
+  /** Creates fraction part for [NumberWithUnits].*/
+  fun createNumberWithUnitsForFraction(number: Fraction, units: List<NumberUnit>):
+    InteractionObject {
+      val numberWithUnits = NumberWithUnits.newBuilder()
+        .addAllUnit(units)
+        .setFraction(number)
+        .build()
+
+      return InteractionObject.newBuilder().setNumberWithUnits(numberWithUnits).build()
+    }
+
+  /** Creates real part for [NumberWithUnits].*/
```

Comment:
Missing space after `.`
`/** Creates real part for [NumberWithUnits]. */`

Reply:
Done